src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -440,6 +487,29 @@ private String getInlineSummary() {
                 .replaceAll("");
     }
 
+    /**
+     * Checks the inline return for forbidden fragments.
```

Comment:
for presence and forbidden fragment.

Reply:
I don't understand, this code does not check for presence, it assumes it. The presence check is in `containsInlineReturnTag`.

Reply:
by this line it looks like only one validation is done
but in  code we do:
```
       if (returnVisible.isEmpty()) {
            log(ast.getLineNumber(), MSG_SUMMARY_JAVADOC_MISSING);
        }
        else if (containsForbiddenFragment(inlineReturn)) {
            log(ast.getLineNumber(), MSG_SUMMARY_JAVADOC);
```

I just hint that better to put all validations description  in javadoc, or update content to be abstract like "do all validations'

Reply:
Ok, I have added a bit to the javadoc to clarify this.